keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/Logistic.java

```diff
@@ -49,19 +61,6 @@ public static Diff dlnPdf(DoubleTensor mu, DoubleTensor s, DoubleTensor x) {
 
         final DoubleTensor dPdb = numeratorPartOne.plus(numeratorPartTwo).divInPlace(denominator).unaryMinusInPlace();
 
-        return new Diff(dPda, dPdb, dPdx);
-    }
-
-    public static class Diff {
-        public final DoubleTensor dPdmu;
-        public final DoubleTensor dPds;
-        public final DoubleTensor dPdx;
-
-        public Diff(DoubleTensor dPdmu, DoubleTensor dPds, DoubleTensor dPdx) {
-            this.dPdmu = dPdmu;
-            this.dPds = dPds;
-            this.dPdx = dPdx;
-        }
+        return ImmutableList.of(dPda, dPdb, dPdx);
```

Comment:
Looks like these were badly named...oops, probably my bad with that.

Can you rename these to: `dPdmu`, `dPds` and `dPdx`?

Reply:
done